src/Components/ServiceScene/Breakdowns/LayoutSwitcher.tsx

```diff
@@ -15,6 +15,11 @@ export interface LayoutSwitcherState extends SceneObjectState {
 }
 
 export type LayoutType = 'single' | 'grid' | 'rows';
```

Comment:
```suggestion 
export type LayoutType = typeof LayoutTypeEnum.single | LayoutTypeEnum.grid | LayoutTypeEnum.rows;
```

Reply:
This would cause all the instance of 'grid' or 'rows' to be updated to the enum type in the codebase. Leaving as is for now
![Screenshot 2025-04-07 at 1 29 36 PM](https://github.com/user-attachments/assets/3ab1df52-34dc-4da5-be4a-88d9f3abd67f)

Reply:
Yeah I think that's good for the codebase tho instead of having these string consts around, I just made the change locally and it doesn't appear to snowball, just 3-4 lines in `FieldValuesBreakdownScene.tsx`, `FieldsAggregatedBreakdownScene.tsx`, `LabelValuesBreakdownScene.tsx`, and `LabelsAggregatedBreakdownScene.tsx`.

But if you want to keep the scope of this PR down we can address in another issue